src/nodes.coffee

```diff
@@ -2236,23 +2235,21 @@ exports.Parens = class Parens extends Base
 
 #### StringWithInterpolations
 
-# Strings with interpolations are in fact just a variation of `Parens` with
-# string concatenation inside.
+exports.StringWithInterpolations = class StringWithInterpolations extends Base
+  constructor: (@body) ->
 
-exports.StringWithInterpolations = class StringWithInterpolations extends Parens
-  # Uncomment the following line in CoffeeScript 2, to allow all interpolated
-  # strings to be output using the ES2015 syntax:
-  # unwrap: -> this
+  #TODO: Is this needed / correct?
```

Comment:
this is correct. This is needed for `traverseChildren` introspection. If you don't use this, `=> "foo #{this}"` will not have its `this` bound because that one traverseChildren wouldn't have traversed inside the interpolations.